driver/src/main.rs

```diff
@@ -175,7 +175,7 @@ struct Options {
     #[structopt(
         long,
         env = "ECONOMIC_VIABILITY_SUBSIDY_FACTOR",
-        default_value = "10.0"
+        default_value = "1.0"
```

Comment:
Before merging, we explicitly set these values in our deployment.

Reply:
If we explicitly set these values in our deployment, doesn't that mean there is nothing to do?

Reply:
Maybe Nick meant *make sure* we set them? I do think we set them but it wouldn't hurt to double check with the staging deployment config.

Reply:
Yes, just want to make sure that changing these defaults won't have unexpected results for the solvers we are currently running.

Reply:
I just checked the deployment and it looks like we set the `ECONOMIC_VIABILITY_SUBSIDY_FACTOR` for each, so nothing to do!

Reply:
> Maybe Nick meant make sure we set them?

:see_no_evil: